AdminUi/src/AdminUi/Controllers/Filters/RedirectAntiforgeryValidationFailedResultFilter.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Backbone.AdminUi.Controllers.Filters;
+
+public class RedirectAntiforgeryValidationFailedResultFilter : IAlwaysRunResultFilter
+{
+    public void OnResultExecuting(ResultExecutingContext context)
+    {
+        if (context.Result is ObjectResult result
+            && result.StatusCode == 400
+            && result.Value is ProblemDetails problemDetails
+            )
+        {
+            problemDetails.Detail = "xsrf-token-may-be-invalid";
```

Comment:
Aren't we hiding some potentially important information by overriding the details?

Reply:
In all my attempts, `problemDetails.Detail` has been null. I can add a check for that, however, if you consider that preferable.

Reply:
Okay, I'm fine with leaving it as it is then.